project/pages/Home/index.jsx

```diff
@@ -0,0 +1,243 @@
+        return (
+
+            <Circle
+                center={coords}
+                pathOptions={{ color: "black", fillColor: "white", fillOpacity: 1 }}
+                radius={20}
+                key={index}
+            >
+                <Popup
+                >
+                    {specificStationName(lineName, index, transportData)}
+                </Popup>
+            </Circle>
+
+        )
+
+    }))
+}
+
+function visualizeLineOnMap(lineName, visibleLineBoolean, transportData, colors, handleLineOnMapClick) {
+    return (
+
+        <>
+            {
+                visibleLineBoolean
+                &&
+                <>
+                    <Polyline
+                        pathOptions={{ color: colors[lineName], weight: 6 }}
+                        positions={specificPolyline(lineName, transportData)}
+                        eventHandlers={{
+                            click: () => handleLineOnMapClick(lineName)
+                        }}
+                    />
+                    {allStationsVisualized(lineName, transportData)}
+                </>
+
+            }
+        </>
+
+    )
+}
+
+function visualizeLineOnList(lineName, visibleLineBoolean, colors) {
+    return (
+        <>
+            {
+                visibleLineBoolean
+                &&
+                <Link
+                    className="w-5/6 h-auto flex flex-col justify-center items-start p-2 text-white font-semibold rounded-sm shadow-2xl"
+                    style={{ backgroundColor: colors[lineName] }}
+                    to={`line/${lineName}`}
+                >
+                    <div>
+                        {lineName}
+                    </div>
+                </Link>
+
+            }
+        </>
+    )
+}
+
+
+const Home = () => {
+
+    const transportData = useSelector((state) => state.data.data)
+    const dataStatus = useSelector(state => state.data.status)
+    const errorData = useSelector(state => state.data.error)
+
+    const colors = useSelector((state) => state.colors.colors)
+    const colorsStatus = useSelector(state => state.colors.status)
+    const errorColors = useSelector(state => state.colors.error)
+
+
+    const navigate = useNavigate()
+
+    // local states
+    const [lineVisible, setLineVisible] = useState({
+        A111: true,
+        A11: true,
+        TB11: true,
+        TM8: true,
+        TM10: true
+    })
+
+    const [isFilterClicked, setIsFilterClicked] = useState({
+        'A': true,
+        'TB': true,
+        'TM': true,
+    })
+
+
+    // Handle visibility of lines on map and in list
+
+    function changeLineVisibility(lineName) {
+        setLineVisible(prevState => ({
+            ...prevState,
+            [lineName]: !prevState[lineName]
+        }))
+    }
+
+    function hideTypeOfTransport(typeOfTrans) {
+        setIsFilterClicked(prevState => ({
+            ...prevState,
+            [typeOfTrans]: !prevState[typeOfTrans]
+        }))
+        const arrBuses = []
```

Comment:
```suggestion
        const buses = []
```
Unnecessary hungarian notation here.